movie/views.py

```diff
@@ -1,3 +1,36 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from .models import Movie
+from analysis.models import Star
+
+class UserFavoriteView(View):
+    def get(self, request):
+        account_id = request.GET.get('id')
+        list_range = request.GET.get('range')
+
+        if not account_id:
+            return JsonResponse({'message': 'KEY_ERROR'}, status=400)
+
+        try:
+            stars = Star.objects.select_related('movie').filter(user_id=int(account_id)).order_by('-point').all()
```

Comment:
```suggestion
            stars = Star.objects.select_related('movie').filter(user_id=int(account_id)).order_by('-point')[:LIMIT]
```
all()은 제거해주시고 LIMIT는 위쪽에 상수로 선언해서 이용해주시기 바랍니다.